recipes-modules/graphviz/graphviz/graphviz-engine-layout.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=graphviz configure layout engine support for dot
+After=warn-boot.timer
+#ConditionFirstBoot=yes
```

Comment:
Actually it should not be commented. 

I put it like this because It wasn't first boot when I was testing it. I will retest with the condition.